castai/resource_castai_eks_cluster.go

```diff
@@ -170,11 +170,6 @@ func resourceCastaiEKSClusterCreate(ctx context.Context, data *schema.ResourceDa
 	}
 
 	clusterID := *resp.JSON200.Id
-	tkn, err := createClusterToken(ctx, client, clusterID)
-	if err != nil {
-		return diag.FromErr(err)
-	}
-	data.Set(FieldEKSClusterToken, tkn)
```

Comment:
IMO we whould remove `FieldEKSClusterToken` from schema also. Now it will be an empty string

Reply:
Of course, good 👁️